include/fuse_common.h

```diff
@@ -358,6 +358,16 @@ struct fuse_loop_config {
  */
 #define FUSE_CAP_NO_OPENDIR_SUPPORT    (1 << 24)
 
+/**
+ *
+ * Indicates support for only invalidate cached pages on explicit request.
+ * If both FUSE_CAP_EXPLICIT_INVAL_DATA and FUSE_CAP_AUTO_INVAL_DATA are set,
+ * then FUSE_CAP_AUTO_INVAL_DATA takes precedence.
```

Comment:
Could you clarify if this refers to the `supported` or `capable` field?

Reply:
Will do. I've pasted an updated comment section below. Please take a look.